src/extension.ts

```diff
@@ -3793,6 +3761,15 @@ export async function activate(context: vscode.ExtensionContext) {
         new HintHoverProvider(treeDataProvider)
       )
     );
+
+    // --- Hints Status Bar Item ---
```

Comment:
Please move the hints status bar item creation to createCmdsStatusBarItems in src/statusBar/index.ts.

Please observer we use a dictionary `statusBarItems: { [key: string]: StatusBarItem }` to keep track of enable/disable state in Espressif tab.